diff --git a/node_modules/bcryptjs/dist/bcrypt.js b/node_modules/bcryptjs/dist/bcrypt.js
index 1b0a648..b81ddfa 100644
--- a/node_modules/bcryptjs/dist/bcrypt.js
+++ b/node_modules/bcryptjs/dist/bcrypt.js
@@ -67,7 +67,7 @@
     function random(len) {
         /* node */ if (typeof module !== 'undefined' && module && module['exports'])
             try {
-                return require("crypto")['randomBytes'](len);
+                return require("crypto-browserify")['randomBytes'](len);
             } catch (e) {}
         /* WCA */ try {
             var a; (self['crypto']||self['msCrypto'])['getRandomValues'](a = new Uint32Array(len));
